src/index.ts

```diff
@@ -0,0 +1,99 @@
+import { getDependencies } from './getdependencies';
+import { getRepoIds } from './getrepos';
+const JupiterOneClient = require('@jupiterone/jupiterone-client-nodejs');
+const gql = require("graphql-tag");
+
+const J1_USER_POOL_ID_PROD = "us-east-2_9fnMVHuxD";
+const J1_CLIENT_ID_PROD = "1hcv141pqth5f49df7o28ngq1u";
+const accessToken = '';
+
+let success = 0, failure = 0;
+const missingDeps = [];
+
+;(async () => {
+  const j1Client = await new JupiterOneClient({
+    account: 'j1dev',
+    username: 'j1dev',
+    password: 'dev',
+    poolId: J1_USER_POOL_ID_PROD,
+    clientId: J1_CLIENT_ID_PROD,
+    accessToken: accessToken,
+    dev: true
+  }).init();
+  const repoMap = await getRepoIds();
+  
+  //value = Repo Name, key = Repo Entity ID
+  for (let value of Array.from(repoMap.values())) {
+    console.log('Repo: ' + value);
+
+    //Get 'From' Repo
+    const mainRepo = await j1Client.queryV1(`FIND CodeRepo WITH name='${value}'`);
+    const depsList = getDependencies(value);
+    if (depsList.length > 0) {
+      for (let i=0; i<depsList.length; i++) {  
+        const fullName = depsList[i].substring(1, depsList[i].indexOf(':'));
+
+        //Check if Bitbucket Repo
+        const depRepo = await j1Client.queryV1(`FIND CodeRepo WITH full_name='${fullName}'`);
```

Comment:
Having to separate BB and GH repos based on the property name (`full_name` vs `name`) seems to me like a sign that the data model for those integrations is too disparate and should be refined! :grimacing:

It'd be pretty epic if you opened a PR to the Bitbucket integration to make it match GitHub. :wink: